quick-start-with-tidb.md

```diff
@@ -6,510 +6,532 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
 
-Other requirements for the target machine include:
+Terminal 1: Write Workload
 
-- The `root` user and its password are required
-- [Stop the firewall service of the target machine](/check-before-deployment.md#check-and-stop-the-firewall-service-of-target-machines), or open the port needed by the TiDB cluster nodes
-- Currently, the TiUP cluster supports deploying TiDB on the x86_64 (AMD64) and ARM architectures:
+Run this command to continuously insert weather readings:
 
-    - It is recommended to use CentOS 7.3 or later versions on AMD64.
-    - It is recommended to use CentOS 7.6 1810 on ARM.
+```shell
+while true; do
+    mysql -h 127.0.0.1 -P 6000 -u root -D sample_weather -vv -e "INSERT INTO readings
+    (station_id, temperature, humidity, pressure) SELECT id as station_id, ROUND(10 + (RAND() *
+    30), 1), ROUND(40 + (RAND() * 60)), ROUND(980 + (RAND() * 40)) FROM stations;"
+    sleep 0.5
+done
+```
 
-### Deploy
+Terminal 2: Read Workload
 
-> **Note:**
+Run this command to continuously query the data:
+
+```shell
+while true; do
+    mysql -h 127.0.0.1 -P 6000 -u root -D sample_weather -e "SELECT s.name, s.location,
+    r.temperature, r.humidity, r.recorded_at FROM stations s JOIN readings r ON s.id = r.station_id
+    WHERE r.recorded_at >= NOW() - INTERVAL 5 MINUTE ORDER BY r.recorded_at DESC LIMIT 5;"
+    sleep 1
+    mysql -h 127.0.0.1 -P 6000 -u root -D sample_weather -e "SELECT s.name,
+    ROUND(AVG(r.temperature), 1) as avg_temp, ROUND(AVG(r.humidity)) as avg_humidity,
+    COUNT(*) as num_readings FROM stations s JOIN readings r ON s.id = r.station_id WHERE
+    r.recorded_at >= NOW() - INTERVAL 1 HOUR GROUP BY s.id, s.name ORDER BY avg_temp DESC;"
+    sleep 3
+done
+```
+
+## Monitor your application
+
+Now that you have a running application with active read and write workloads, let's use TiDB's built-in monitoring tools to understand what's happening.
+
+### TiDB Dashboard: SQL insights
+
+Access TiDB Dashboard by checking its URL on the terminal you are running TiUP playground (typically http://127.0.0.1:2379/dashboard - user: `root`, no password) to:
+
+1. View your running queries:
+
+    - Go to "SQL Statements" to see your weather data queries
+    - Check execution times, and additional optional columns, and click on the statement for more execution details
+    - Go to "Slow Queries" to identify queries that might need optimization
+
+2. Monitor instance status:
+
+    - Check "Overview" and "Cluster Info" for general topology, and resource usage
+    - Check "Monitoring" to see the database load and other relevant metrics
+    - Check "Search Logs" for any query errors
+
+### Grafana: system performance
+
+Access Grafana by checking its URL on the terminal you are running TiUP playground (typically http://127.0.0.1:3000 - user: `admin`, password: `admin`). Change the password or Skip. You can visualize the built-in dashboards by clicking on the Search icon and the folder "Test-Cluster":
```

Comment:
"typically http://127.0.0.1:2379/dashboard" is confusing here because this section assumes TiProxy is added. In that case, " http://127.0.0.1:60838" is also typical.